WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -151,4 +151,12 @@ extension URL {
         components?.queryItems = queryItems
         return components?.url ?? self
     }
+
+    /// Gravatar doesn't support "Cache-Control: none" header. So we add a random query parameter to
+    /// bypass the backend cache and get the latest image.
+    func appendingGravatarCacheBusterParam() -> URL {
+        var urlComponents = URLComponents(url: self, resolvingAgainstBaseURL: false)
+        urlComponents?.queryItems?.append(.init(name: "_", value: "\(NSDate().timeIntervalSince1970)"))
```

Comment:
Do you mind adding a simple unit test for this function? This function does not work if there is no query in the original URL.

```
> URL(string: "https://wordpress.com")!.appendingGravatarCacheBusterParam()
$R0: Foundation.URL = "https://wordpress.com"
```